src/components/Button.js

```diff
@@ -24,6 +27,10 @@ const styles = StyleSheet.create({
     color: colors.lightestText,
     fontFamily: 'titillium-web-bold'
   },
+  buttonStyle: {
+    width: '60%',
+    alignSelf: 'center'
```

Comment:
we like attributes ordered alphabetically, right?

Reply:
right!